ledger-tool/src/bigtable.rs

```diff
@@ -574,17 +587,36 @@ pub fn bigtable_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
     // again resulting in the default value. the arg having declared a
     // `default_value()` obviates `is_present(...)` tests since they will always
     // return true. so we consede and compare against the expected default. :/
-    let (subcommand, sub_matches) = matches.subcommand();
+    let (_, sub_matches) = matches.subcommand();
```

Comment:
Could we pass in `sub_matches` so we don't have to do this three times?

Reply:
Yep! Done.